packages/angular_devkit/schematics_cli/bin/schematics.ts

```diff
@@ -354,34 +338,48 @@ function getUsage(): string {
 
 /** Parse the command line. */
 const booleanArgs = [
-  'allowPrivate',
   'allow-private',
   'debug',
   'dry-run',
-  'dryRun',
   'force',
   'help',
   'list-schematics',
-  'listSchematics',
   'verbose',
   'interactive',
 ];
 
-function parseArgs(args: string[] | undefined): minimist.ParsedArgs {
-  return minimist(args, {
+/** Parse the command line. */
+function parseArgs(args: string[]): { _: string[] } & Record<string, unknown> {
+  const { $0, _, ...options } = yargsParser(args, {
     boolean: booleanArgs,
-    alias: {
-      'dryRun': 'dry-run',
-      'listSchematics': 'list-schematics',
-      'allowPrivate': 'allow-private',
-    },
     default: {
       'interactive': true,
       'debug': null,
-      'dryRun': null,
+      'dry-run': null,
+    },
+    configuration: {
+      'unknown-options-as-args': false,
```

Comment:
It looks like `unknown-options-as-args` is false by default.
https://github.com/yargs/yargs-parser/tree/0fdbfa1fe960f76d520f75437e97c79727804908#unknown-options-as-args

Reply:
Yeah, left there to be more explicate, but I can remove it if you wish.